### ISSUE TITLE
Why not control the nb_epoch within the fit function?

### ISSUE THREAD

**Issue by eshijia** [maciejkula/triplet_recommendations_keras#5]
The codes have following lines:

```
for epoch in range(num_epochs)
    ...
        model.fit(...,nb_epoch=1, ...)
    ...
```

Why not set the nb_epoch as num_epochs directly?

**Comment by maciejkula** [comment]
This is to run the validation on the test set after every epoch. You could do it via callbacks but this is just less work.
